libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -1327,6 +1400,11 @@ void WaveClip::ShiftBy(double delta) noexcept
 // We are within the clip if start <= t < end.
 // Note that BeforeClip and AfterClip must be consistent
 // with this definition.
+bool WaveClip::SplitsPlayRegion(double t) const
```

Comment:
This function follows a comment that used to be attached to the following function.